Mathlib/RingTheory/DedekindDomain/IntegralClosure.lean

```diff
@@ -247,9 +253,44 @@ instance integralClosure.isDedekindDomain_fractionRing [IsDedekindDomain A] :
     IsDedekindDomain (integralClosure A L) :=
   integralClosure.isDedekindDomain A (FractionRing A) L
 
+open nonZeroDivisors Algebra IsLocalization IsFractionRing IsScalarTower in
```

Comment:
This proof became quite long but now `IsIntegrallyClosed C` is not needed anymore. It should be moved since it has nothing to do with dedekind domains, but I don't know where.

Reply:
What does `#min_imports` say?

Reply:
```lean
import Mathlib.LinearAlgebra.BilinearForm.DualLattice
import Mathlib.LinearAlgebra.FreeModule.PID
import Mathlib.RingTheory.DedekindDomain.Basic
import Mathlib.RingTheory.RingHom.Finite
import Mathlib.RingTheory.Trace.Basic
```
I am not against creating a new file.